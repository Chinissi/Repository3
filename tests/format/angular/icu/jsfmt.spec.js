// The ICU component can be moved to the angular/angular tests when ICU parsing
// with control flow syntax has been fixed, but until then this component will
// not work with {angularControlFlowSyntax: true} (the default).
run_spec(import.meta, ["angular"], { angularControlFlowSyntax: false });
