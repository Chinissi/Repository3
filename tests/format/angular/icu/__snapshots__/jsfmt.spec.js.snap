// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icu.component.html - {"angularControlFlowSyntax":false} format 1`] = `
====================================options=====================================
angularControlFlowSyntax: false
parsers: ["angular"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<span i18n>Updated {minutes, plural, =0 {just now} =1 {one minute ago} other {{{minutes}} minutes ago}}</span>
<span i18n>The author is {gender, select, male {male} female {female} other {other}}</span>
<span i18n>Updated: {minutes, plural, =0 {just now} =1 {one minute ago} other {{{minutes}} minutes ago by {gender, select, male {male} female {female} other {other}}}}</span>

=====================================output=====================================
<span i18n
  >Updated {minutes, plural, =0 {just now} =1 {one minute ago} other
  {{{minutes}} minutes ago}}</span
>
<span i18n
  >The author is {gender, select, male {male} female {female} other
  {other}}</span
>
<span i18n
  >Updated: {minutes, plural, =0 {just now} =1 {one minute ago} other
  {{{minutes}} minutes ago by {gender, select, male {male} female {female} other
  {other}}}}</span
>

================================================================================
`;
